### ISSUE TITLE
Doesn't correctly handle nested quotes

### ISSUE THREAD

**Issue by chocolateboy** [evanlucas/argsplit#2]
### test

``` javascript
let argsplit = require('argsplit');
let string = `--title "Peter's Friends" --name 'Phil "The Power" Taylor'`;

console.log(argsplit(string));
```
### expect

``` javascript
[ "--title", "Peter's Friends", "--name", 'Phil "The Power" Taylor' ]
```
### get

``` javascript
[ "--title", '"Peter\'s Friends" --name \'Phil "The Power"', "Taylor'" ]
```

**Comment by evanlucas** [comment]
yep, that's a bug. Will try to get a fix out today-ish.

**Comment by evanlucas** [comment]
Fixed in 7cf6344ab9e2fac57abf5f251a150d3001fc8fcb

**Comment by evanlucas** [comment]
Also published in v1.0.2
